biothings/web/settings/default.py

```diff
@@ -104,32 +104,34 @@
     'dotfield': {'type': bool, 'default': False},
     '_sorted': {'type': bool, 'default': True},  # alaphabetically
     'always_list': {'type': list, 'max': 1000},
-    'allow_null': {'type': list, 'max': 1000}
+    'allow_null': {'type': list, 'max': 1000},
 }
 ANNOTATION_KWARGS = {
     '*': COMMON_KWARGS.copy(),
     'GET': {'id': {'type': str, 'path': 0, 'required': True}},
-    'POST': {'id': {'type': list, 'max': 1000, 'required': True, 'alias': 'ids'}}
+    'POST': {'id': {'type': list, 'max': 1000, 'required': True, 'alias': 'ids'}},
 }
 QUERY_KWARGS = {
     '*': COMMON_KWARGS.copy(),
     'GET': {
         'q': {'type': str, 'default': None},
         'aggs': {'type': list, 'max': 1000, 'alias': 'facets'},
+        'post_filter': {'type': list, 'max': 1000},
```

Comment:
I know the original implementation set `type` as list, but I feel it does not need to, a string should be good enough. What do you think?

Reply:
I think the `post_filter` can work with string type, but still need to double check it.

Reply:
Yes, we can change `post_filter` to str type, and it still works.

Reply:
@newgene

Reply:
ok, let's change it to string then

Reply:
It already updated. :)